projects/skia/skia.diff

```diff
@@ -360,6 +360,45 @@ index 48954a8938..5a4f354a21 100644
      Builder builder;
      for (int i = 0; i < count; i++) {
          LayerInfo info;
+diff --git a/src/gpu/GrPathUtils.cpp b/src/gpu/GrPathUtils.cpp
+index 74a53e9e09..872b28ced6 100644
+--- a/src/gpu/GrPathUtils.cpp
++++ b/src/gpu/GrPathUtils.cpp
+@@ -394,7 +394,11 @@ void convert_noninflect_cubic_to_quads(const SkPoint p[4],
+                                        SkPathPriv::FirstDirection dir,
+                                        SkTArray<SkPoint, true>* quads,
+                                        int sublevel = 0) {
+-
++    #if defined(IS_FUZZING)
++        if (sublevel >= 7) {
++            return;
++        }
++    #endif
+     // Notation: Point a is always p[0]. Point b is p[1] unless p[1] == p[0], in which case it is
+     // p[2]. Point d is always p[3]. Point c is p[2] unless p[2] == p[3], in which case it is p[1].
+ 
+diff --git a/src/ports/SkDebug_stdio.cpp b/src/ports/SkDebug_stdio.cpp
+index ec4e3fec77..e2c1a48521 100644
+--- a/src/ports/SkDebug_stdio.cpp
++++ b/src/ports/SkDebug_stdio.cpp
+@@ -12,9 +12,13 @@
+ #include <stdio.h>
+ 
+ void SkDebugf(const char format[], ...) {
+-    va_list args;
+-    va_start(args, format);
+-    vfprintf(stderr, format, args);
+-    va_end(args);
++    #if !defined(IS_FUZZING)
```

Comment:
There are a lot of SkDebugf calls in Skia's GPU code, I did this to silence them.

Reply:
Ack